testing/test_assertrewrite.py

```diff
@@ -1955,6 +1955,25 @@ def test_simple_failure():
             assert hook.find_spec("file") is not None
             assert self.find_spec_calls == ["file"]
 
+    def test_assert_excluded_rootpath(
```

Comment:
for due diligence we probably need some positive/negative tests against python packages with/without pytest plugins that are installed